cmd/operator-sdk/internal/genutil/genutil.go

```diff
@@ -22,9 +22,10 @@ import (
 	"path/filepath"
 	"strings"
 
+	"github.com/spf13/viper"
```

Comment:
Not a big deal but can you separate all non-`github.com/operator-framework/operator-sdk` imports below with a space between them and the last `github.com/operator-framework/operator-sdk` import:

```go
...

"github.com/operator-framework/operator-sdk/internal/pkg/scaffold"
"github.com/operator-framework/operator-sdk/internal/util/projutil"
flags "github.com/operator-framework/operator-sdk/cmd/operator-sdk/internal"

log "github.com/sirupsen/logrus"
"github.com/spf13/viper"
```

We've been following that convention to make large import blocks less confusing.

Reply:
Aha! I had a sense there was a method there. I'll go back and clean everything up.